.gitignore

```diff
@@ -16,3 +16,6 @@ build/
 
 # The . idea folder (hidden on OS X) in the solution root contains IntelliJ's project specific settings files.
 .idea/
+
+# Nothing in here for now
+static/
```

Comment:
im not sure if we want to keep the `static/` folder in .gitignore bc we might add custom css files and might want to track those changes

Reply:
Yeah, I just wasn't sure how you guys were going to go about CSS so i put it in .gitignore for now, but I can change that right now!

Reply:
Yeah can you take out static from gitignore? We're gonna need that folder